### PR TITLE
Servlet to delete a group.

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes issue #314 

Changes: Added API end point to delete a particular group. 
test locally at -> http://127.0.0.1:4000/aaa/deleteGroup.json?group=groupName

### CODE REVIEW

**Comment by DravitLochan** [comment]
@saurabhjn76 @dynamitechetan @jyoth1raditya @Orbiter Please review!

**Comment by DravitLochan** [comment]
@mariobehling Please review and merge!

File: src/ai/susi/server/api/aaa/DeleteGroupService.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    @Override
+    public ServiceResponse serviceImpl(Query post, HttpServletResponse response, Authorization rights, JsonObjectWithDefault permissions) throws APIException {
+        JSONObject result = new JSONObject();
+        String groupName = post.get("group", null);
+
+        if( groupName!=null ) {
+            if( DAO.group.has(groupName)) {
+                DAO.group.remove(groupName);
+                result.put("message", "Group successfully deleted");
```
Comment (Orbiter) [inline src/ai/susi/server/api/aaa/DeleteGroupService.java]: add `result.put("accepted", true);`

File: src/ai/susi/server/api/aaa/DeleteGroupService.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            } else {
+                result.put("message","Group does not exist");
```
Comment (Orbiter) [inline src/ai/susi/server/api/aaa/DeleteGroupService.java]: add `result.put("accepted", false);`

File: src/ai/susi/server/api/aaa/DeleteGroupService.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            }
+            return new ServiceResponse(result);
+        } else {
+            result.put("message", "Bad call, group name parameter not specified");
+        }
```
Comment (Orbiter) [inline src/ai/susi/server/api/aaa/DeleteGroupService.java]: add `result.put("accepted", false);`

**Comment by Orbiter** [comment]
My review was done in the context of this new issue https://github.com/fossasia/susi_server/issues/326 which should make handling of servlets more consistent.

**Comment by DravitLochan** [comment]
Cool. Will follow it from now!

**Comment by DravitLochan** [comment]
@Orbiter Please approve if no other changes are there. 
Thanks!

File: src/ai/susi/server/api/aaa/DeleteGroupService.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    @Override
+    public ServiceResponse serviceImpl(Query post, HttpServletResponse response, Authorization rights, JsonObjectWithDefault permissions) throws APIException {
+        JSONObject result = new JSONObject();
+        String groupName = post.get("group", null);
+
+        if( groupName!=null ) {
+            if( DAO.group.has(groupName)) {
+                DAO.group.remove(groupName);
+                result.put("accepted", "true");
```
Comment (Orbiter) [inline src/ai/susi/server/api/aaa/DeleteGroupService.java]: arrrrrrrrrrrrr
`result.put("accepted", true);`
not
`result.put("accepted", "true");`

**Pull request merged**